lang/MoarVM/Portfile

```diff
@@ -48,6 +48,11 @@ configure.args      --cc=${configure.cc} \
 configure.cflags-append \
                     -I${prefix}/include/libtommath
 
+# The dylib machinery uses -rpath, not available on Tiger
+platform darwin 8 {
+    patchfiles-append patch-build-setup.diff
```

Comment:
Please rename the patch file to make it more clear its only relevant for tiger (and older ?..)

Reply:
Fixed

Reply:
Thanks..